### ISSUE TITLE
Project directory not getting built on composer install

### ISSUE THREAD

**Issue by vaibhav-kaushal** [qcubed/qcubed#1130]
While QCubed 3.0 was in beta, composer installation created the `project` directory. It is not happening in the final release.

**Comment by olegabr** [comment]
it may be related to the #1131. please, review and merge the PR #1132.

**Comment by vaibhav-kaushal** [comment]
Done. Trying reinstallation.

**Comment by vaibhav-kaushal** [comment]
Failed. The merge had no effect on installation. Does it download the framework from packagist? Because the commit ID which is downloaded is the one that is present on packagist while the one on current master is different. 

Look here: https://packagist.org/packages/qcubed/qcubed

**Comment by vaibhav-kaushal** [comment]
I have added a webhook to qcubed main repo to ensure that packagist updates our code everytime we update master.

**Comment by olegabr** [comment]
I've made a manual update on the `packagist` for the `qcubed/qcubed` repo. now it should take the latest commit from composer.

**Comment by vaibhav-kaushal** [comment]
Let me try again then.

**Comment by vaibhav-kaushal** [comment]
IT worked!

**Comment by Benxxy** [comment]
Can u please explain step by step how to fix this problem

**Comment by kukrik** [comment]
It was the second problem, we corrected our names, folders, etc... It is irrelevant...

If I understand correctly, you want to install qcubed to your computer? The first step is to follow the instructions: https://github.com/qcubed/app-starter

If you have any questions, feel free to ask!

**Comment by Benxxy** [comment]
Yes I want to install it on my computer and before two days using composer it creates vendor and project files, but now I get only vendor file and then I have problems with my qcubed.inc.php.

**Comment by kukrik** [comment]
Please specify operation system for your computer!

I use a MacBook and Homebrew :). Homebrew is intended for as a virtual server.

When the Homebrew is made, it will go to whatever "Sites" or "www" or another folder. Of course, to configure before :(.

Then you can begin to pull into qcubed.

PS. My English is poor and very bad, because I am so sorry...

By the way - I use the command line: sudo composer create-project --stability=dev qcubed/app-starter your_dir_name. But before to go to the folder: cd /location/folder/folder/etc...

**Comment by vaibhav-kaushal** [comment]
@Benxxy Can you detail out the steps and where it fails? THe directory structure when it fails? What are your problems with qcubed.inc.php?

**Comment by spekary** [comment]
This is now fixed. Please test again.
